wlauto/workloads/androbench/__init__.py

```diff
@@ -0,0 +1,41 @@
+#    Copyright 2013-2015 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import os
+
+from wlauto import AndroidUiAutoBenchmark
+
+
+class Androbench(AndroidUiAutoBenchmark):
+    name = 'androbench'
+    description = """Androbench measures the storage performance of device"""
+    package = 'com.andromeda.androbench2'
+    activity = '.main'
+    run_timeout = 10 * 60
+
+    def update_result(self, context):
+        super(Androbench, self).update_result(context)
+        db = '/data/data/com.andromeda.androbench2/databases/history.db'
+        qs = 'select * from history'
+        res = 'results.raw'
+        os.system('adb shell sqlite3 %s "%s" > %s' % (db, qs, res))
+        fhresults = open("results.raw", "rb")
+        results = fhresults.readlines()[0].split('|')
```

Comment:
As per comments in the perivous pull request, please do not invoke either adb or sqilte3 in the code. *all* interactions with the device must be performed via `self.device`; and you should not assume the presence of sqlite3 binary either on the host or on the target device. Also, do not create files in the current working directory; files on the host must be relative to `context.output_directory`,  `context.run_output_directory` or `tempfile.gettempdir()` (for temporary files not intended to be included in the results).

As outlined previously, you can get history.db from the device with

     host_results = os.path.join(context.output_directory, 'results.db')
     self.device.pull_file(db, host_results, as_root=True)

You can then use the built-in Python module (https://docs.python.org/2/library/sqlite3.html) to query the database to extract the metrics.